applications/ecallmgr/src/ecallmgr_fs_route.erl

```diff
@@ -347,6 +347,9 @@ start_message_handling(_Node, _FetchId, CallId, JObj) ->
 
 -spec route_req(ne_binary(), ne_binary(), wh_proplist(), atom()) -> wh_proplist().
 route_req(CallId, FetchId, Props, Node) ->
+    SwitchURL = ecallmgr_fs_node:sip_url(Node),
+    [_, SwitchURIHost] = binary:split(SwitchURL, <<"@">>),
```

Comment:
Why is this extraction not wrapped in a try/catch when the ecallmgr_fs_event_stream's version is?